Add Funders Talk event (Josh Greenberg) (#1029)
USRSE/usrse.github.io: _events/2023/2023-01-funder-talk-series.md

---
title: Joshua Greenberg - Alfred P. Sloan Foundation
subtitle: US-RSE Funders Talk Series
expires: 2023-01-27

> event_date: "January 26, 2023"

Reviewer: ```suggestion
event_date: "February 2, 2023"
```

layout: event
duration: 60
repeated: false
category: funder
time:

>     - - start: 2023-01-26T21:00:00Z

Reviewer: ```suggestion
    - - start: 2023-02-02T21:00:00Z
```

        end: 2023-01-26T22:00:00Z
---

## US-RSE Funders Talk Series Info

US-RSE is happy to be continuing its series of conversations with funders for 1) RSEs to learn about funding opportunities and review and other processes, 2) RSEs to ask questions of the funder, and 3) funders to ask questions and learn about RSEs.  

Visit the [Funder Talk Series]({{ site.baseurl }}/wg/outreach/#funder-series) for more information and links to past presentations.

## Event

> This talk in the US-RSE Funder Talk Series will feature Dr. Joshua Greenberg, Program Director of the Alfred P. Sloan Foundation's Digital Information Technology program. This event will take place **Thursday, January 26th at 4 PM ET, 3 PM CT, 2 PM MT, 1 PM PT**

Reviewer: ```suggestion
This talk in the US-RSE Funder Talk Series will feature Dr. Joshua Greenberg, Program Director of the Alfred P. Sloan Foundation's Digital Information Technology program. This event will take place **Thursday, February 2nd at 4 PM ET, 3 PM CT, 2 PM MT, 1 PM PT**
```
